src/resource/file.ts

```diff
@@ -200,23 +201,8 @@ type SaveFileOptions = WriteFileOptions & {
  * @param options Additional parameters for file creation (e.g. a slug).
  * @returns A Promise that resolves to the saved file, if available, or `null` if the current user does not have Read access to the newly-saved file. It rejects if saving fails.
  */
-export async function saveFileInContainer<FileExt extends File | NodeFile>(
-  folderUrl: Url | UrlString,
-  file: FileExt,
-  options?: Partial<SaveFileOptions>,
-): Promise<FileExt & WithResourceInfo>;
-/**
- * @deprecated `saveFileInContainer` should only have `File` input
- */
-export async function saveFileInContainer<
-  FileExt extends File | NodeFile | Buffer,
->(
-  folderUrl: Url | UrlString,
-  file: FileExt,
-  options?: Partial<SaveFileOptions>,
-): Promise<FileExt & WithResourceInfo>;
 export async function saveFileInContainer<
-  FileExt extends File | NodeFile | Buffer,
+  FileExt extends File | BlobFile | NodeFile,
```

Comment:
What is the first `File` here?

Reply:
That's the one globally defined by typescript

Reply:
Even though the constructor doesn't work till mode 20, the type seems to exist in node 18